pages/linux/asterisk.md

```diff
@@ -0,0 +1,29 @@
+# asterisk
+
+> Telephone and exchange (phone) server.
+> This executable is used for running the server itself, and managing an already running instance.
```

Comment:
Perhaps this will read better?
```suggestion
> Used for running the server itself, and managing an already running instance.
```

Reply:
My only concern with that change would be possible confusion with other related executables such as `rasterisk`

Reply:
Ah, I see. `asterisk` and `rasterisk` should be documented on different pages, so I'm not sure that would be an issue?